cmd/scollector/collectors/stream.go

```diff
@@ -0,0 +1,82 @@
+package collectors
+
+import (
+	"reflect"
+	"runtime"
+	"sync"
+	"time"
+
+	"bosun.org/collect"
+	"bosun.org/metadata"
+	"bosun.org/opentsdb"
+)
+
+type StreamCollector struct {
```

Comment:
The main reason for it is to be able to generate scollector.collector.count for this collector.